README.md

```diff
@@ -33,9 +33,9 @@ You can find our documentation [here](https://skypilot.readthedocs.io/en/latest/
 
 ## Example SkyPilot Task
 
-A SkyPilot task is specified as a YAML file containing the resource requirements, data to be synced, setup commands, and the task commands. 
+A SkyPilot task is specified as a YAML file containing: resource requirements, data to be synced, setup commands, and the task commands. 
 
-Once written in a YAML, the task can be launched on any available cloud. Example:
+Once written in this [**unified interface**](https://skypilot.readthedocs.io/en/latest/reference/yaml-spec.html), the task can be launched on any available cloud. Example:
```

Comment:
Should we re-emphasize that this indeed a YAML, not some complex interface? e.g. `[**unified YAML interface**]`

Reply:
Maybe we can list both possible interface, like `Once written in this [**unified interface**](https://skypilot.readthedocs.io/en/latest/reference/yaml-spec.html) (YAML or Python API),`

Reply:
Good call, done.